app/build.gradle

```diff
@@ -144,4 +144,7 @@ dependencies {
     implementation "com.squareup.okhttp3:logging-interceptor:5.0.0-alpha.5"
     implementation "com.squareup.okhttp3:okhttp:5.0.0-alpha.5"
 
+    testImplementation 'junit:junit:4.+'
+
+    androidTestImplementation 'com.squareup.okhttp3:mockwebserver:5.0.0-alpha.5'
```

Comment:
Small nitpick, but just noticed this is an alpha version of the library...that the best option?

Reply:
I had to use that because Kurt used the alpha version for his okhttp retrofit work and the libraries had to match